src/main/java/com/github/steveice10/mc/protocol/data/game/entity/metadata/MetadataType.java

```diff
@@ -158,11 +153,11 @@ public static MetadataType<?> read(ByteBuf in, MinecraftCodecHelper helper) thro
 
         return VALUES.get(id);
     }
-    
+
     public static MetadataType<?> from(int id) {
         return VALUES.get(id);
     }
-    
+
     public static int size() {
```

Comment:
may as well undo this when you undo the wildcard imports since there won't be any other changes to this class

Reply:
I changed the import stuff. Might as well fix wrong spaces in the code. Noone enforces a code style except when it's in PRs by others lol

Reply:
> Noone enforces a code style except when it's in PRs by others lol

If you're talking about wildcard imports, I assure you that almost every active contributor avoids using them. 
If you're talking about the whitespace, it has nothing to do with code style. It's about introducing unnecessary diff, and a commit, to a file that has nothing to do with the PR.

Reply:
Is there any list of allowed wildcard imports? Intellij idea by default wildcards java.something and another one. Wouldn't Lombok also be a good fit?
Imo tabs and whitespaces are part of the code style and are unnecessary bytes stored in the file while giving no benefit. That's why linters line spotless exist.
I am more wondering why it got introduced in the first place. Would you want code style fixes in a separate PR?

Reply:
> Is there any list of allowed wildcard imports?

Test assertions are okay

> Intellij idea by default wildcards java.something and another one

IIRC the GUI libs AWT and Swing

> Imo wrongly placed tabs and whitespaces are part of the code style and are unnecessary bytes stored in the file while giving no benefit.

I agree and am completely okay with the style change itself, I just meant that the reason I brought it up is that trivial style changes, and that file itself, aren't relevant to this PR.

> I am more wondering why it got introduced in the first place. Would you want code style fixes in a separate PR?

This codebase is quite old. 
If there are broad code style changes you'd like to apply, they should be done as a separate PR. 
Typically though, for things that aren't terribly important, I would personally prefer fixing things on a case by case basis. For example, potentially adding spotless and using their ratchet feature.